src/app/order/history/history.component.ts

```diff
@@ -1,7 +1,7 @@
 import { Component, OnInit } from '@angular/core';
 import { OrderService, Order } from '../order.service';
 import { Config } from '../../restaurant/restaurant.service';
-import * as io from 'socket.io-client';
+import io from 'socket.io-client';
```

Comment:
non-blocking

Is this related to turning on esModuleInterop in tsconfig.json?

Reply:
Yes